app/domain/authentication/authn_jwt/signing_key/create_signing_key_interface.rb

```diff
@@ -0,0 +1,75 @@
+module Authentication
+  module AuthnJwt
+    module SigningKey
+      # Factory that returns the interface implementation of FetchSigningKey
+      CreateSigningKeyInterface ||= CommandClass.new(
+        dependencies: {
+          fetch_provider_uri: Authentication::AuthnJwt::SigningKey::FetchProviderUriSigningKey,
+          fetch_jwks_uri: Authentication::AuthnJwt::SigningKey::FetchJwksUriSigningKey,
+          logger: Rails.logger
+        },
+        inputs: %i[authentication_parameters]
+      ) do
+        def call
+          create
+        end
+
+        private
+
+        def create
```

Comment:
please change this to create_signing_key`

Reply:
Done